### PR TITLE
Update error message in run command

### PR DESCRIPTION
**Pull request by hemikak**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/19550

## Check List 
- [X] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java [hunk 1]
```diff
@@ -218,8 +218,8 @@ public void execute() {
             targetPath = sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.",
+                    "invalid Ballerina source path, it should either be a module name in a Ballerina project, a " +
```
Comment (MaryamZi) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: do we need the "in a Ballerina project" part?
```suggestion
                    "invalid Ballerina source path, it should either be a module name, a " +
```

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java [hunk 2]
```diff
@@ -218,8 +218,8 @@ public void execute() {
...
+                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension or a .jar file.",
```
Comment (MaryamZi) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: I believe this should be
```suggestion
                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension, or a .jar file.",
```

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 1 above)
Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: This error message seems to be consistent with all other commands. IMO stating 'in a Ballerina project' would be good for a newcomer who is experimenting.

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 2 above)
Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: @praneesha I am a bit confused here. What would be the right way ?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 1 above)
Comment (MaryamZi) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: Shall we rephrase this as follows then 
```suggestion
                    "invalid Ballerina source path, it should either be a name of a module in a Ballerina project, a " +
```

Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: +1 , i'll make that change across all the commands.

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 2 above)
Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: I changed it as:
> "invalid Ballerina source path, it should be a name of a module in a Ballerina project or a file with a '.bal' extension or an executable .jar file."

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 1 above)
Comment (praneesha) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: +1

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 2 above)
Comment (praneesha) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: Shall we reword it like this: "invalid Ballerina source path. It should either be a name of a module in a Ballerina project, a file with a '.bal' extension, or an executable .jar file."

Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: Fixed

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java (hunk 1 above)
Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: Fixed

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java [hunk 3]
```diff
@@ -218,9 +218,10 @@ public void execute() {
             targetPath = sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.",
-                    "ballerina run {<bal-file> | <module-name>}",
+                    "invalid Ballerina source path. It should either be a name of a module in a Ballerina project, " +
+                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension, or an executable .jar " +
```
Comment (MaryamZi) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: .bal and .jar are inconsistent here right?

```suggestion
                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension, or an executable '.jar' " +
```

Comment (hemikak) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java]: Fixed

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/DocCommand.java [hunk 4]
```diff
@@ -250,8 +250,8 @@ public void execute() {
             targetPath = this.sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension. use the -a or --all " +
+                    "invalid Ballerina source path. It should either be a name of a module in a Ballerina project or " +
+                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension. use the -a or --all " +
```
Comment (MaryamZi) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/DocCommand.java]: Build seems to use uppercase U for use. Should this be the same?

```suggestion
                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension. Use the -a or --all " +
```
